cartridges/int_adyen_SFRA/cartridge/client/default/js/checkout/checkout.js

```diff
@@ -17,7 +17,9 @@ module.exports = {
                     data.options
                 );
             });
-            adyenCheckout.methods.renderGenericComponent();
+            var currentStage = location.search.substring(location.search.indexOf("=")+1);
+            if(currentStage === "shipping")
```

Comment:
We have to keep in mind that virtual products don't have a shipping stage, but we do have to render the payment methods. We must think of a solution for that.